### PR TITLE
Panel page design

### PR DESCRIPTION
**Pull request by mesparza5**
Updating styling to match comp.

### CODE REVIEW

**Comment by KorvinSzanto** [comment]
I think it'd be okay to disable `selector-max-id` and `selector-max-compound-selectors` in `_shared.scss` for now, you can do that with `/* stylelint-disable selector-max-id, selector-max-compound-selectors */`

We should consider allowing more id selectors and compound selectors at a time repo wide.

File: assets/cms/scss/panels/_shared.scss [hunk 1]
```diff
@@ -440,6 +452,64 @@ div.ccm-panel-detail-form-actions {
...
+
+  &.side-panel {
+    padding: 10px;
+  }
+}
+
+/**
+ * Page Design Side Panel
+ */
+#ccm-panel-page-design {
```
Comment (aembler) [inline assets/cms/scss/panels/_shared.scss:464]: We should be putting these rules in a separate scss file. For example we have versions SCSS in `panels/pages/_versions.scss`, let's put this in `panels/pages/_design.scss`

File: assets/cms/scss/_toolbar.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /* stylelint-disable selector-max-compound-selectors, property-no-vendor-prefix, selector-max-id */
 div#ccm-toolbar {
   background: #fafafe;
+  background: #fafafe;
```
[the remark this answers is not in the document]
Comment (mesparza5) [reply assets/cms/scss/_toolbar.scss]: I removed both, due to we already have `background-color: $gray-100;` in place

File: assets/cms/scss/panels/_shared.scss [hunk 3]
```diff
@@ -440,6 +452,64 @@ div.ccm-panel-detail-form-actions {
   padding: 15px 20px 15px 340px;
   position: fixed;
   width: 100%;
+
+  &.side-panel {
```
[the remark this answers is not in the document]
Comment (mesparza5) [reply assets/cms/scss/panels/_shared.scss]: This was removed.

**Pull request merged**